be/src/runtime/result_sink.cpp

```diff
@@ -35,6 +36,17 @@ ResultSink::ResultSink(const RowDescriptor& row_desc, const std::vector<TExpr>&
     : _row_desc(row_desc),
       _t_output_expr(t_output_expr),
       _buf_size(buffer_size) {
+
+    if (!sink.__isset.type || sink.type == TResultSinkType::MYSQL_PROTOCAL) {
```

Comment:
|| sink.type == TResultSinkType::MYSQL_PROTOCAL is unnecessary

Reply:
`sink.type` is a newly added field of `TResultSink`.
So for the request from the old FE planner, this field is not set.
And for the request from the new FE planner, this field may be set to `MYSQL_PROTOCAL`.

Reply:
It's better to add a comments here